packages/core/components/NestedDropdown/nesteddropdown.styles.css

```diff
@@ -0,0 +1,81 @@
+.nested-dropdown {
+  li {
+    list-style: none;
+    cursor: pointer;
+
+    &:focus {
+      border: 2px solid orange;
+    }
+  }
+
+  .searchInput {
+    border: none;
+    position: relative;
+    display: inline-block;
+    width: 100%;
+  }
+
+  .nested-dropdown__option {
+    padding: 7px 15px;
+    background: white;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    width: 325px;
+    cursor: pointer;
+  }
+
+  .secondTierValue {
+    display: inline-block;
+  }
+
+  .selectableItem:hover {
+    background-color: #ccc;
+  }
+
+  & > div {
+    position: relative;
+    & > span {
+      position: absolute;
+      font-size: 10px;
+      top: 4px;
+      right: 12px;
+      padding: 9px;
+      background: #fff;
+      pointer-events: none;
+    }
+  }
+
+  & > ul {
+    max-height: 375px;
+    overflow-y: scroll;
+    position: absolute;
+    z-index: 3;
+    width: 375px;
+
+    & > li {
+      font-weight: bold;
+
+      & > ul > li {
+        position: relative;
+        font-weight: normal;
+        white-space: nowrap;
+        & > span {
+          position: absolute;
+          left: -20px;
+        }
+      }
+    }
+  }
+
+  .nested-dropdown__group--hidden > ul {
```

Comment:
could this be changed to .nested-dropdown.group.hidden? idk if maybe there's some naming collision here you're trying to avoid? but it should already be cascaded under .nested-dropdown

Reply:
why not use the .hide class below?

Reply:
Removed the underscores and double dashes. 
Changed the name to nested-dropdown-group-collapsed. This is needed for the Arrow functions logic. The logic is checking the className of the li that houses the ul. If it is .nested-dropdown-group-collapsed, then it is collapsing the corresponding ul.